Sources/Apollo/ApolloClient.swift

```diff
@@ -154,6 +153,124 @@ extension ApolloClient: ApolloClientProtocol {
                                       callbackQueue: queue,
                                       completionHandler: resultHandler)
   }
+
+#if swift(>=5.5.2) && canImport(_Concurrency)
```

Comment:
I'm not super clear on if `canImport(_Concurrency)` is necessary. I saw Vapor doing this for their async/await rollout.